internal/docker/builder.go

```diff
@@ -124,16 +124,16 @@ func (d *Builder) removeImages() error {
 			d.log("Not cleaning up image with tags: %v", img.RepoTags)
 			continue
 		}
-		bprintName := img.Labels["complement_blueprint"]
+		contextStr := img.Labels[complementLabel]
 		keep := false
 		for _, keepBprint := range d.Config.KeepBlueprints {
-			if bprintName == keepBprint {
+			if contextStr == keepBprint {
```

Comment:
We could add in some regex pattern matching here to support keeping all of the homeservers in a blueprint, `fed.perf_many_messages.*`